### ISSUE TITLE
Pokémon mini core?

### ISSUE THREAD

**Issue by noddy360** [TASEmulators/BizHawk#3162]
I want to suggest a core for the obscure Pokémon mini handheld.
PokeMini would probably be the emulator of choice for this: https://sourceforge.net/projects/pokemini/

I tested the Libretro core on BizHawk and it ran the couple of games I tested just fine. I managed to use savestates and rewind without too many issues. I even managed to record a movie using this core. At first it desynced due to the initial recording skipping the date/time settings screen for some reason, but I managed to fix it by using TAStudio, which also seemed to work just fine. I couldn't test if the reed switch works, though.

In theory it wouldn't be too diffucult to port to BH properly, at least compared to other emulators that have been ported in the past.

**Comment by Spikestuff** [comment]
GBE+ also has it available as under their [source code](https://github.com/shonumi/gbe-plus) only for the sake of noting and is also still under active development.

**Comment by noddy360** [comment]
GBE+ 1.6 released in early April which adds the Pokemon mini core in a "Largely complete" state and with IR multiplayer support.
I haven't personally tested this emulator myself though, so I can't say for certain if it's really better than PokeMini in terms of accuracy or ease of use yet.

**Comment by CasualPokePlayer** [comment]
GBE+ has a poor frontend/core separatation and SDL slapped into the core so it wouldn't be easy to port over.

**Comment by noddy360** [comment]
Edited the title and the description to be more in line with more recent core requests.

**Comment by Jhynjhiruu** [comment]
PokeMini probably isn't the best choice of emulator for this; it's got several known inaccuracies, which don't come up when playing games, but could cause issues with TASing - the most notable being that it doesn't emulate interrupt priorities accurately. [Minimon](https://github.com/asterick/minimon.js/) might be a better choice; it's got pretty good emulation/UI separation (there's an existing Libretro core already, as well as a [web interface](https://minimon.scyl.us/)), is more accurate than PokeMIni, and is designed around the modern understanding of the system.

**Comment by CasualPokePlayer** [comment]
Right, I don't think that emulator looks like quite the best idea when it decides to put a ton of Pokemon ROMs directly in its repo.

**Comment by Jhynjhiruu** [comment]
It is, however, currently the best emulator for accuracy

**Comment by noddy360** [comment]
I have updated the first post including some new testing with the libretro core as well as testing PokeMini (the SDL version) on Linux (through WSL) and libTAS. Not much more progress on my research other than this.

I do have a question though. Would waterboxing PokeMini be possible? Or perhaps is there another emulator that I'm missing that would be better for that?
